docs/content/guides/traffic_management/configuration_validation/admission_control.md

```diff
@@ -61,7 +62,167 @@ Configure the validating admission webhook to reject invalid Gloo custom resourc
       {{< /notice >}}
+      
+## Enable full Envoy validation (beta)
+
+In addition to strict resource validation, you can enable full Envoy validation in your Gloo Gateway setup. The full Envoy validation adds another validation layer to the validation webhook by converting the translated xDS snapshot into static bootstrap configuration that can be fed into Envoy. This way, you can validate configuration that is typically accepted by Gloo Gateway, but later rejected by Envoy. For example, you might have a transformation policy in your VirtualService that uses an invalid Inja template. Gloo Gateway cannot validate the Inja template and therefore accepts the configuration. However, with the full Envoy validation enabled, this configuration is checked against Envoy and rejected if Envoy detects invalid configuration. 
+
+{{% notice note %}}
+The full Envoy validation is a beta feature. 
+{{% /notice %}}
+
+1. Follow the [Hello World guide]({{% versioned_link_path fromRoot="/guides/traffic_management/hello_world/" %}}) to set up the hello world app and expose it with a VirtualService.
+2. Edit the Settings to enable strict resource validation without the full Envoy validation. Make sure to also set `disableTransformationValidation: true` to disable the transformation validation. To persist this setting between Gloo Gateway upgrades, add this setting to your Helm values file instead. 
+   ```sh
+   kubectl edit settings default -n gloo-system
+   ```
+   
+   Enter the following values: 
+   ```yaml
+   ...
+   spec:
+     gateway:
+       validation:
+         allowWarnings: false
+         alwaysAccept: false
+         disableTransformationValidation: true
+         fullEnvoyValidation: false
+   ...
+   ```
+   
+3. Add a transformation policy to the hello world VirtualService that uses an invalid Inja template. The following example does not close the else statement. Verify that this configuration is accepted by Gloo Gateway. 
+   {{< highlight yaml "hl_lines=23" >}}
+kubectl apply -f- <<EOF
+apiVersion: gateway.solo.io/v1
+kind: VirtualService
+metadata:
+  name: default
+  namespace: gloo-system
+spec:
+  virtualHost:
+    domains:
+    - '*'
+    routes:
+    - matchers:
+      - exact: /all-pets
+      options:
+        prefixRewrite: /api/pets
+        stagedTransformations:
+          regular: 
+            responseTransforms: 
+            - responseTransformation: 
+                transformationTemplate: 
+                  headers: 
+                    test_header: 
+                      text: '{% if default(data.error.message, "") != %}400{% else '
+      routeAction:
+        single:
+          upstream:
+            name: default-petstore-8080
+            namespace: gloo-system
+EOF
+   {{< /highlight >}}
+   
+4. Review the logs of the `gateway-proxy` pod. Verify that although Gloo Gateway accepted the configuration, an error regarding the malformatted Inja template is reported by Envoy. 
```

Comment:
maybe say `warning` since the log level is called warning, not error
```suggestion
4. Review the logs of the `gateway-proxy` pod. Verify that although Gloo Gateway accepted the configuration, a warning regarding the malformatted Inja template is reported by Envoy. 
```